### DISCUSSION TITLE
TESLAUSB: json file from "USB"

### DISCUSSION THREAD

**Discussion by alberpao** [marcone/teslausb#888] [Ideas]
Hi Marcone,
I did installed the Teslausb and work fair good (just some issues on missing upload seems more connected with the poor WiFi connection in garage than a problem on your code ;) ).
A question: recently I discover in a forum that data from Dashcam (and I assume also from Sentry) are georeferences using the json file generated by the car with the video: any chance to upload it to my NAS (as I do for video) together with the video themself?
Thanks
Paolo

**Comment by marcone** [comment]
> recently I discover in a forum that data from Dashcam (and I assume also from Sentry) are georeferences using the json file generated by the car with the video

The json file is already archived/uploaded, however note that only Sentry and Saved recordings have an associated json file, but not Dashcam.

**Comment by alberpao** [comment]
Thanks Marcone!!! Appreciated
I'll check, I see at the moment the video.
Paolo
